### PR TITLE
Adding custom compatibility error message in pdfRender

### LINKED ISSUE #1803: PDF reader with a more graceful message?
## Summary

During testing with UNICEF, the following has come up:

> I tried to see the content of the lessons. When I choose one of them then a message came up, that the inline browser of the app cannot open pdf file (see attached photo). If I understood correctly that means that the app cannot open pdf file.

image

Since it's a tablet, it won't be optimal to have the student navigate to a separate application to view the PDF.

Does a solution exist to guide the user to enable inline PDF viewing?

If not, please feel free to close this issue. I actually think the text is pretty okay as it is tbh.

Otherwise the text could say "please download the PDF and open it with an external application for displaying PDFs. In case you have one installed already, it may open up when you click this link:"

### PR DESCRIPTION
**Pull request by DXCanas**
Also:
- More semantic $trNameSpace
- Adding a name to the component for easier debugging
- Using a computed prop inside of the component for url to reduce snake case in JS

## Issues addressed

resolves #1803 

screenshot_2017-07-19-23-59-52

### CODE REVIEW

**Comment by DXCanas** [comment]
Because of string freeze, likely going to have to be punted to 0.6, so targeting at develop

File: kolibri/plugins/document_pdf_render/assets/src/views/index.vue [hunk 1]
```diff
@@ -24,7 +36,20 @@
   import PDFobject from 'pdfobject';
   import ScreenFull from 'screenfull';
   import iconButton from 'kolibri.coreVue.components.iconButton';
+
+  const pdfObjectOptions = {
+    fallbackLink: false,
+  };
+
   export default {
+    $trNameSpace: 'pdfDisplayPage',
```
Comment (rtibbles) [inline kolibri/plugins/document_pdf_render/assets/src/views/index.vue]: Note that $trNameSpace is no longer used, the name property is used instead.

File: kolibri/plugins/document_pdf_render/assets/src/views/index.vue [hunk 2]
```diff
@@ -24,7 +36,20 @@
...
+    $trs: {
+      pdfCompatibilityError: 'PDFs cannot be displayed inside Kolibri on your device',
+      pdfDownloadLink: 'Click here download the PDF for viewing in a third-party application',
```
Comment (benjaoming) [inline kolibri/plugins/document_pdf_render/assets/src/views/index.vue]: Seems to be missing a "to".

Are these strings translatable?

Comment (rtibbles) [reply kolibri/plugins/document_pdf_render/assets/src/views/index.vue]: Yes, this is our convention for translatable strings in the frontend :)

Comment (benjaoming) [reply kolibri/plugins/document_pdf_render/assets/src/views/index.vue]: haha, I thought it was something about "table rows"

**Pull request merged**